DFC.Digital/DFC.Digital.Web.Sitefinity.Core/SitefinityExtensions/RecycleBinService.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Autofac;
+using Autofac.Integration.Mvc;
+using DFC.Digital.Data.Interfaces;
+using DFC.Digital.Web.Sitefinity.Core.Interfaces;
+using System;
+using Telerik.Sitefinity.Security.Claims;
+using Telerik.Sitefinity.Services;
+
+namespace DFC.Digital.Web.Sitefinity.Core.SitefinityExtensions
+{
+    [System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverage]
```

Comment:
Is it possible to move the system manager check down one level, so that at least this module can be included in the code coverage?

Reply:
changed